src/public/app/widgets/type_widgets/mind_map.ts

```diff
@@ -5,6 +5,9 @@ import nodeMenu from "@mind-elixir/node-menu";
 import type FNote from "../../entities/fnote.js";
 import type { EventData } from "../../components/app_context.js";
 
+// allow node-menu plugin css to be bundled by webpack
+import "@mind-elixir/node-menu/dist/style.css";
```

Comment:
Since we dynamically import the library in order not to download everything at once unless it's needed, shouldn't we do the same here?

Reply:
I just double checked with a local electron-forge build – nothing we need to here anymore:
the CSS is bundled in the same JS file as the other mind-elixir code ­– and it only gets "downloaded", when you browse to a mind-map note.

![grafik](https://github.com/user-attachments/assets/f0d15781-1075-46fd-92d0-3e59ad013105)

[1] I was browsing to an excalidraw note type
[2] is where I browsed to a mind map note type
[3] you can see the CSS being embedded in the mind map chunk